### PR TITLE
feat(skin): update design tokens from mistica-design branch: production

### PR DESCRIPTION
**Pull request by tuentisre**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

**Comment by amegias** [comment]
any owner here? any reason for these changes?

File: Sources/MisticaCommon/Colors/BlauColorPalette.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@ import UIKit
 struct BlauColors: MisticaColors {
     static let palette = BlauColorPalette()
 
-    let backgroundBrand = MisticaColor.solid(BlauColors.palette.blauBluePrimary | BlauColors.palette.darkModeBlack)
+    let backgroundBrand = BlauColors.palette.blauBluePrimary | BlauColors.palette.darkModeBlack
```
Comment (alexanegon) [inline Sources/MisticaCommon/Colors/BlauColorPalette.swift:14]:
```suggestion
    let backgroundBrand = MisticaColor.solid(BlauColors.palette.blauBluePrimary | BlauColors.palette.darkModeBlack)
```

File: Sources/MisticaCommon/Colors/ColorToolkit+UIColor.swift [hunk 2]
```diff
@@ -653,7 +658,7 @@ public extension UIColor {
 public extension BrandStyle {
     var preferredStatusBarStyle: UIStatusBarStyle {
         switch self {
-        case .movistar, .vivo, .o2, .o2New, .blau, .custom, .vivoNew, .telefonica, .tu:
+        case .movistar, .vivo, .o2, .o2New, .blau, .custom, .vivoNew, .telefonica:
```
Comment (alexanegon) [inline Sources/MisticaCommon/Colors/ColorToolkit+UIColor.swift:661]: Please do not erase `tu` brand:
```suggestion
        case .movistar, .vivo, .o2, .o2New, .blau, .custom, .vivoNew, .telefonica, .tu:
```

File: Sources/MisticaCommon/Colors/MisticaColors.swift [hunk 3]
```diff
@@ -11,7 +11,7 @@ import UIKit
 public protocol MisticaColors {
     var background: UIColor { get }
     var backgroundAlternative: UIColor { get }
-    var backgroundBrand: MisticaColor { get }
+    var backgroundBrand: UIColor { get }
```
Comment (alexanegon) [inline Sources/MisticaCommon/Colors/MisticaColors.swift:14]:
```suggestion
    var backgroundBrand: MisticaColor { get }
```

**Review by alexanegon — CHANGES_REQUESTED** [review]

File: Sources/MisticaCommon/Colors/ColorToolkit+UIColor.swift (hunk 2 above)
Comment (yceballost) [reply Sources/MisticaCommon/Colors/ColorToolkit+UIColor.swift:661]: This PR was automatically generated by tokens automation 😬

**Pull request closed without merging**